### ISSUE TITLE
Netbeans lockup while linting

### ISSUE THREAD

**Issue by Mardonis** [luka-zitnik/jshint-for-netbeans#4]
Is there a way to see a log if the JSHint plugin (while linting) has caused a crash somehow? I had copied over some old javascript code to a new js file. I then did a format to align everything correctly. There was a few extra [return] rows that i wanted to delete. I happened to hit the delete button on one of the empty lines while it was saying that is was linting and it has locked up netbeans. OR of there isa way to do a debug while im working so i can send you the debug info?

Thnk
Mardonis

**Comment by luka-zitnik** [comment]
All unhandled errors are reported directly to user, so there are no 
special logs to go through.

I believe debugging this would be a bad experience for the both of us. 
Mostly because I would have to base any effort on very week assumptions. 
From this standpoint, anything could be causing the trouble, another 
module or even a buggy version of java.

**Comment by Mardonis** [comment]
Sorry for the delay. I was finally able to grab the error when all I did was open a js file the right click and hit format. Which it should format the js file correctly. The file only had about 1800 lines of code which iv formatted larger files before. Upon doing that I looked at the bottom and it said it was linting then a red error came up. Clicking it brought up the following error any ideas?  After grabbing the error it still says linting file with progress bar going even thought it crashed and had not linted the file. When also trying to close out of the linting process it then decides to lockup. Need anymore help just let me know. Thank you.

java.lang.IllegalStateException
    at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3291)
    at org.mozilla.javascript.gen.jshint_js_1.call(jshint.js)
    at lukazitnik.jshint.JSHint.callJSHint(JSHint.java:183)
    at lukazitnik.jshint.JSHint.lint(JSHint.java:101)
    at lukazitnik.jshint.JSHint.lint(JSHint.java:62)
[catch] at lukazitnik.jshint.annotations.LintingThread.run(LintingThread.java:39)
